tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\ExistingSite;
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\search_api\Functional\ExampleContentTrait;
+use Drupal\user\Entity\Role;
+
+/**
+ * Class to test List pages content type on existing site tests.
+ */
+class ListPagesTest extends ShowcaseExistingSiteTestBase {
+
+  use ExampleContentTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Create admin user.
+    $role = Role::load($this->createRole([]));
+    $role->setIsAdmin(TRUE);
+    $role->save();
+    $user = $this->createUser([]);
+    $user->addRole($role->id());
+    $user->save();
+    $this->drupalLogin($user);
+  }
+
+  /**
+   * Check creation List pages content through the UI.
+   */
+  public function testCreateListPages() {
+    // Mark test content for deletion after the test has finished.
+    $this->markEntityTypeForCleanup('node');
+
+    $assert_session = $this->assertSession();
+    $page = $this->getSession()->getPage();
+
+    // Create some test nodes.
+    $day = 00;
+    for ($i = 0; $i < 12; $i++) {
+      $date = new DrupalDateTime('20-10-2020');
+      $values = [
+        'title' => 'News number ' . $i,
+        'type' => 'oe_sc_news',
+        'body' => 'This is content number ' . $i,
+        'language' => 'en',
+        'status' => NodeInterface::PUBLISHED,
+        'oe_publication_date' => [
+          'value' => '2022-04-' . $day++,
```

Comment:
Now I see what you are trying to achieve with the `00` above :) but it won't work as you expect.
https://3v4l.org/uSOmV
https://stackoverflow.com/a/1699980/246724
https://3v4l.org/2lRtG

```suggestion
          'value' => sprintf('2022-04-%02d', $i + 1),
```